### PR TITLE
Upgrade devDependencies

### PR DESCRIPTION
**Pull request by nc-kreiserm**
#### Version change

We use [Semantic Versioning 2.0](https://semver.org/)

- [ ] Major - Non-backwards compatible changes
- [ ] Minor - Backwards compatible changes
- [ ] Patch - Backwards-compatible bug fix
- [ ] None - Internal changes

#### Procedural Steps

All commits have been signed-off for the Developer Certificate of Origin. See https://github.com/NextCapital/maybe/blob/main/CONTRIBUTING.md#dco-sign-off-methods

#### **Notes:**

TODO: Fill in

### CODE REVIEW

File: .cspellcache [hunk 1]
```diff
@@ -0,0 +1 @@
...spell-ext.json","bff29ea09b5c57f6757c322f20d67fcb","/Users/kreiserm/code/maybe/node_modules/@cspell/dict-ruby/cspell-ext.json","74d0696fcb9f992fb856074ec89719db","/Users/kreiserm/code/maybe/node_modules/@cspell/dict-rust/cspell-ext.json","937ff9d941ee0ae8f6dea4c9d2f6841e","/Users/kreiserm/code/maybe/node_modules/@cspell/dict-scala/cspell-ext.json","8f01fa0d8a8a7bcddab6f83a9ec2fa69","/Users/kreiserm/code/maybe/node_modules/@cspell/dict-software-terms/cspell-ext.json","d3372fee6368ff61b7da6c69ca4405af","/Users/kreiserm/code/maybe/node_modules/@cspell/dict-software-terms/softwareTerms.txt.gz","6bb1caa5593c02ac850704f07cc23275","/Users/kreiserm/code/maybe/node_modules/@cspell/dict-sql/cspell-ext.json","82d9bdfcc2921cba28f3cd47ad30e8bf","/Users/kreiserm/code/maybe/node_modules/@cspell/dict-swift/cspell-ext.json","da83d48b4d10342a919b0c008ccfd3df","/Users/kreiserm/code/maybe/node_modules/@cspell/dict-typescript/cspell-ext.json","77b7d528ba25587efe833e11704122d6","/Users/kreiserm/code/maybe/node_modules/@cspell/dict-typescript/typescript.txt.gz","aa2e606ead099a84fe955107801ab301","/Users/kreiserm/code/maybe/node_modules/@cspell/dict-vue/cspell-ext.json","60c4486f8b2f52d948fab125597233d7"]
```
Comment (NC-piercej) [inline .cspellcache:1]: be sure to add this to `.gitignore`

**Review by NC-piercej — COMMENTED** [review]
seems fine after nitpicks

File: .cspellcache (hunk 1 above)
Comment (nc-kreiserm) [reply .cspellcache]: ope

**Review by NC-piercej — APPROVED** [review]
🚀 be sure to do a patch version

**Pull request merged**